### ISSUE TITLE
[INFO] What is the main difference between multivariate and multiple time series?

### ISSUE THREAD

**Issue by guilhermeparreira** [unit8co/darts#2146]
Thank you guys for the amazing package!

I read a lot on Darts' documentation those days, but I am still in doubt:

+ What is the main difference under the hood between **multiple** and **multivariate** time series?

Do you recommend any books/articles/videos that provide an overview of this topic?

**Context**

My understanding so far:

+ **Multivariate** and **multiple** use only one single model for all series;

Questions:

+ In **Multiple** time series, if I don't use static covariates, would I get the same results as univariate? (as they are treated independently?)
+ In **Multiple** series you can use static covariates to introduce correlation between the series. How is the correlation modeled? What is the main difference compared to **multivariate** time series?
+ In **Multivariate** time series, it is modeled all together. How do univariate `scikit-learn` models adapt to this scenario?

I know that I have a lot of questions, but I would like to know more about this topic...
Thank you in advance for your patience.

**Comment by madtoinou** [comment]
Hi @guilhermeparreira,

You can find some information about the difference between the two [here](https://unit8co.github.io/darts/userguide/timeseries.html#multivariate-time-series-vs-multiple-time-series)  and [there](https://unit8co.github.io/darts/userguide/forecasting_overview.html#support-for-multivariate-series).

Briefly, multivariate is used to qualify  the dimensions of a series. It means that it has several columns ("opposite" of univariate) for one time index. In order to forecast this kind of series, the model needs to be able to generate multiple outputs.

Multiple series, on the other hand, is a way to train models : it means that several series, of different length and non-overlapping time indexes, are used to train a single model (an example [here](https://unit8co.github.io/darts/examples/01-multi-time-series-and-covariates.html). It can be used to for example increase the size of the training dataset, perform meta-learning or capture patterns over non-overlapping series.

To answer you questions:
- You won't, even if the multiple series just correspond to a unique series being sliced into several series as some training samples won't be generated into the multiple series scenario due to border effects. When multiple series are used, the tabularization generates a single pair of arrays (`X`, `y`) based on all the series (concatenating all the training samples) : series are not treated independently.
- It depends on the model but you can consider the static covariates as an additional feature. With a multivariate series, all the others columns as considered as features but the static covariate does not bring any information.
- Either the `sk-learn` model supports multi outputs or it's wrapped in the `MultiOutputRegressor` class.

In summary, these two terms correspond to two different notions and are kind-of specific to Darts. I hope that it clarifies it.

**Comment by guilhermeparreira** [comment]
Thank you very much for the clarification!

I thought that these were two consolidated ways to make the forecast...

But your very precise answer gave me good insights.

Thank you again
